drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -149,6 +149,38 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
     joins(:bed_nights).where(s_t[:date_provided].eq(date))
   end
 
+  # @param project [Project]
+  # @param range [DateRange]
+  # enrollments that conflict with an entry/exit date
+  # * entry date on exit date is allowed
+  # * multiple entry dates on same day are not allowed
+  scope :with_conflicting_dates, ->(project:, range:) do
+    entry_date = range.begin
+    raise unless entry_date
+
+    scope = where(project_id: project.project_id, data_source: project.data_source_id)
```

Comment:
This needs to take into account conflicting enrollments that are WIP. The below works (tho its a poorly named scope, I think to match the Filter conventions)
```suggestion
    scope = with_project([project.id])
```